test/cql-pytest/util.py

```diff
@@ -69,3 +74,213 @@ def new_test_table(cql, keyspace, schema, extra=""):
+
+    def create_timestamp_item(self):
+        self.items_count += 1
+        return datetime.now()
+
+    def create_uuid1_item(self):
+        self.items_count += 1
+        return uuid.uuid1()
+
+    def create_tiny_int_item(self):
+        self.items_count += 1
+        return self._int_number(low=self.TINY_INT_LOW_RANGE, high=self.TINY_INT_HIGH_RANGE)
+
+    def create_uuid_item(self):
+        self.items_count += 1
+        return uuid.uuid4()
+
+    def create_varchar_item(self):
+        self.items_count += 1
+        return self._random_string(string_size=random.randint(10, self.SMALL_INT_HIGH_RANGE))
+
+    def create_multiple_items(self, mode, num_of_items):
+        if mode == CassandraTypes.ASCII:
+            return [self.create_random_ascii_item() for _ in range(num_of_items)]
+        elif mode == CassandraTypes.BIGINT:
+            return [self.create_bigint_item() for _ in range(num_of_items)]
+        elif mode == CassandraTypes.BLOB:
+            return [self.create_blob_item() for _ in range(num_of_items)]
+        elif mode == CassandraTypes.BOOLEAN:
+            return [self.create_bool_item() for _ in range(num_of_items)]
+        elif mode == CassandraTypes.DATE:
+            return [self.create_date_item() for _ in range(num_of_items)]
+        elif mode == CassandraTypes.DOUBLE:
+            return [self.create_double_item() for _ in range(num_of_items)]
+        elif mode == CassandraTypes.DECIMAL:
+            return [self.create_decimal_item() for _ in range(num_of_items)]
+        elif mode == CassandraTypes.FLOAT:
+            return [self.create_float_item() for _ in range(num_of_items)]
+        elif mode == CassandraTypes.INET:
+            return [self.create_ip_address() for _ in range(num_of_items)]
+        elif mode == CassandraTypes.INT:
+            return [self.create_int_item() for _ in range(num_of_items)]
+        elif mode == CassandraTypes.SMALLINT:
+            return [self.create_small_int_item() for _ in range(num_of_items)]
+        elif mode == CassandraTypes.TEXT:
+            return [self.create_text_item() for _ in range(num_of_items)]
+        elif mode == CassandraTypes.TIME:
+            return [self.create_time_item() for _ in range(num_of_items)]
+        elif mode == CassandraTypes.TIMESTAMP:
+            return [self.create_timestamp_item() for _ in range(num_of_items)]
+        elif mode == CassandraTypes.TIMEUUID:
+            return [self.create_uuid1_item() for _ in range(num_of_items)]
+        elif mode == CassandraTypes.TINYINT:
+            return [self.create_tiny_int_item() for _ in range(num_of_items)]
+        elif mode == CassandraTypes.UUID:
+            return [self.create_uuid_item() for _ in range(num_of_items)]
+        elif mode == CassandraTypes.VARCHAR:
+            return [self.create_varchar_item() for _ in range(num_of_items)]
+        raise TypeError(f"The following type '{mode}' not supported")
+
+    def create_multiple_lines(self, num_of_lines, modes):
```

Comment:
In this class you use the terms "items" and "lines" in confusing way. Instead of "item" it probably be "value". Instead of "lines" it should be "rows"?